admin/spec/generator/solidus_admin/component_generator_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails/version'
+require 'rails/generators'
+require 'rails/generators/app_base'
+require "rails/generators/testing/behaviour"
+require "rails/generators/testing/setup_and_teardown"
+require "rails/generators/testing/assertions"
+require 'generators/solidus_admin/component/component_generator'
+
+RSpec.describe SolidusAdmin::ComponentGenerator, type: :generator do
+  include Rails::Generators::Testing::Behaviour
+  include Rails::Generators::Testing::SetupAndTeardown
```

Comment:
Looking at [railties/lib/rails/generators/testing/setup_and_teardown.rb](https://github.com/rails/rails/blob/34bd524a9d7580ceecbf4a6dd9dced6296e20f97/railties/lib/rails/generators/testing/setup_and_teardown.rb) it appears that this `include` may not be necessary.

Reply:
Thanks fixed, I also moved the generic includes and requires to the spec_helper.rb